src/task1/orchestrator/mainWindowLayout.py

```diff
@@ -0,0 +1,137 @@
+from PyQt5 import QtWidgets
+
+def drawHome(window):
+    """
+    Draws the home layout
+
+    :param window: The QT session
+    :return:
+    """
+
+    # Quit Button
+    btn = QtWidgets.QPushButton("Quit", window)
+    btn.clicked.connect(window.exitProgram)
+    btn.resize(200, 50)
+    btn.move(730, 490)
+
+    # Step 1 Label
+    bl = QtWidgets.QLabel(window)
+    bl.setText("Step 1 -  select the source image folders")
+    bl.resize(600, 50)
+    bl.move(25, 20)
+    bl.setStyleSheet('color: darkblue')
+
+    # Step 2 Label
+    bl = QtWidgets.QLabel(window)
+    bl.setText("Step 2 -  Run the modules. Keep in mind to run them in order if applicable")
+    bl.resize(600, 50)
+    bl.move(25, 200)
+    bl.setStyleSheet('color: darkblue')
+
+    # Select Directory Curr Directory
+    window.dirLabel = QtWidgets.QLabel(window)
+    window.dirLabel.setText(window.dir)
+    window.dirLabel.resize(800, 50)
+    window.dirLabel.move(230, 100)
+
+    # Select Directory Button
+    btn = QtWidgets.QPushButton("Browse For Photo Directory", window)
+    btn.clicked.connect(window.getDirectory)
+    btn.resize(200, 50)
+    btn.move(25, 90)
+
+def drawTask1(window):
+    """
+    Draws the home layout
+
+    :param window: The QT session
+    :return:
+    """
+    # Module 1 Button
+    window.autofilterButton = QtWidgets.QPushButton("Module 1 - Auto Filter", window)
+    window.autofilterButton.clicked.connect(window.launchAutoFilterModule)
+    window.autofilterButton.move(25, 250)
```

Comment:
Does pyqt have layout classes?

Reply:
It does but we are way too deep into this to refactor :(